src/language/variables.md

```diff
@@ -38,6 +38,48 @@ String name = 'Bob';
   of using `var`, rather than type annotations, for local variables.
 {{site.alert.end}}
 
+## Null safety
```

Comment:
@munificent : I rewrote this section. Maybe I should move it to the overview. Let me know your thoughts.